meta-phosphor/common/recipes-kernel/linux/linux-obmc_4.2.bb

```diff
@@ -14,7 +14,7 @@ SRC_URI += "file://obmc-bsp.scc \
 LINUX_VERSION ?= "4.2"
 LINUX_VERSION_EXTENSION ?= "-openbmc-${SRCPV}"
 
-SRCREV="${AUTOREV}"
+SRCREV="0e0ca76818c0e81aa7647c7cc9d06b6babaacd77"
```

Comment:
Let's use tags instead of revisions; we're often rebasing our patches so commits may dissapear.

I've created and pushed tag openbmc-20151028-1 (https://github.com/openbmc/linux/releases/tag/openbmc-20151028-1) for you to use. This corresponds to 0e0ca76818c0e81aa7647c7cc9d06b6babaacd77 (https://github.com/openbmc/linux/commit/0e0ca76818c0e81aa7647c7cc9d06b6babaacd77)